eng/deploy.yaml

```diff
@@ -279,6 +279,7 @@ stages:
       displayName: Install Correct .NET Version
       inputs:
         useGlobalJson: true
+        workingDirectory: $(Build.SourcesDirectory)
```

Comment:
How does this change anything? This is the default no?

Reply:
The default is the the workspace directory, and I'm not sure where the task is searching first that it ends up finding the test one. I'd rather set this to a static version if at all possible.

Reply:
if we set it to a static version then we have to remember to update 2 places rather than 1 when we need to update the version. I don't think that is a good path forward.

Reply:
Then are you ok with trying this fix?

Reply:
I see the bad global.json that is causing this. And i don't think this change will fix it. We should just put a real version in there. Is there a specific reason it needs to be a nonexistent version?